openzaak-alfresco-share/src/main/resources/alfresco/web-extension/messages/openzaak-alfresco-share.properties

```
@@ -0,0 +1,95 @@
+# Type labels
+type.drc_zaaktypefolder=Openzaak zaaktype folder
+type.drc_zaakfolder=Openzaak zaakfolder
+
+type.drc_document=Openzaak document
+type.drc_gebruiksrechten=Openzaak gebruiksrechten
+type.drc_oio=Openzaak oio
+
+# Case type messages
+form.advancedsearch.zaaktype.label=Zaaktype zoeken
+form.advancedsearch.zaaktype.description=Zoeken naar Openzaak zaaktypes
+
+form.zaak.set.zaaktypeinformatie.label=Zaaktype informatie
+form.zaak.set.zaaktypeinformatie.zaaktypeidentificatie.label=Zaaktype identificatie
+form.zaak.set.zaaktypeinformatie.zaaktypeurl.label=Zaaktype url
+
+# Case messages
+form.advancedsearch.zaak.label=Zaak zoeken
+form.advancedsearch.zaak.description=Zoeken naar Openzaak zaken
+
+form.zaak.set.openzaak.label=Zaak informatie
+form.zaak.set.openzaak.zaakurl.label=Zaak url
+form.zaak.set.openzaak.zaaktypeurl.label=Zaaktype url
+form.zaak.set.openzaak.zaakidentificatie.label=Zaak identificatie
+form.zaak.set.openzaak.zaakbronorganisatie.label=Zaak bronorganisatie
+
+# Case document messages
+form.advancedsearch.document.label=Document zoeken
+form.advancedsearch.document.description=Zoeken naar Openzaak documenten
+
+form.document.set.basisinformatie.label=Basis informatie
+form.document.set.basisinformatie.uuid.label=Document UUID
+form.document.set.basisinformatie.bestandsnaam.label=Bestandsnaam
+form.document.set.basisinformatie.identificatie.label=Identificatie
+form.document.set.basisinformatie.titel.label=Titel
+form.document.set.basisinformatie.beschrijving.label=Beschrijving
+form.document.set.basisinformatie.informatieobjecttype.label=Informatie object type
```

```suggestion
form.document.set.basisinformatie.informatieobjecttype.label=Informatieobjecttype
```

Could also use "documenttype" which is less abstract for end users.